docs/gemstones/rpaste.md

```diff
@@ -0,0 +1,79 @@
+---
+title: rpaste - Pastebin Tool
+author: Steven Spencer
+contributors:
+tags:
+  - rpaste
+  - Mattermost
+  - pastebin
+---
+
+# Introduction to `rpaste`
+
+`rpaste` is a tool for sharing code, log output, and other extra long text. It is a pastebin created by the Rocky Linux developers. It is particularly useful when you need to share something publicly, but do not want to dominate the feed with your text. This is extremely important when using Mattermost, which has bridges to other IRC services. It can be installed on any Rocky Linux system. If your desktop machine isn't Rocky Linux or if you simply do not want to install the tool, you can use it manually, by accessing the [pinnwand URL](https://rpa.st)and then paste the system output or text that you want to share. `rpaste` allows you to create this information automatically.
+
+## Installation
+
+Installation of `rpaste` on Rocky Linux:
+
+```
+sudo dnf --enablerepo=extras install rpaste
+```
+
+## Uses
+
+For major system problems, you may need to send all of your system's information so that it can be reviewed for problems. To do this run:
+
+```
+rpaste --sysinfo
+```
+
+Which, will return the link to the pinnwand page:
+
+```bash
+Uploading...
+Paste URL:   https://rpa.st/2GIQ
+Raw URL:     https://rpa.st/raw/2GIQ
+Removal URL: https://rpa.st/remove/YBWRFULDFCGTTJ4ASNLQ6UAQTA
+```
+
+You can then review the information yourself in a browser and decide if you want to keep it or remove it and start over. If you want to keep it, you can copy the "Paste URL" and share that with whomever you are working with, or in the feed on Mattermost. To remove, simply copy the "Removal URL" and open that in your browser.
```

Comment:
is it supposed to be "Removal" or "Remove"?

Reply:
Since the auto-generated link is the "Removal URL", then to properly reference what was returned by the command, I figured we needed to quote it and use that. (for reference, see the bash section above those two sentences.)